### DISCUSSION TITLE
find: zscore-normalize: No such file or directory

### DISCUSSION THREAD

**Discussion by Jkgarciam** [jcreinhold/intensity-normalization#83] [Q&A]
Hi, I recently changed laptop and now i cant run the intensity normalization package from the terminal. 

macOS Sonoma 14.4.1 
2.6 GHz 6-Core Intel Core i7
Radeon Pro 560X 4 GB
Intel UHD Graphics 630 1536 MB
16 GB 2400 MHz DDR4

**Comment by jcreinhold** [comment]
Can you import `intensity_normalization` in the Python REPL in which it's installed? If not, you probably need to activate the virtual environment in which you installed it.

If that's not the case, please provide the following information:

 - intensity-normalization version (e.g., 0.3.8):
 - numpy version (e.g., 1.0):
 - OS (e.g., Linux):
 - How you installed intensity-normalization (`conda`, `pip`, source):
 - Build command you used (if installed from source):
 - Python version:

**Comment by Jkgarciam** [comment]
Yes, I can import intensity_normalization in python REPL

Intensity_normalization version 2.2.4, installed with pip
numpy version  1.26.4
macOS Sonoma 14.4.1
python version 3.9

**Reply by jcreinhold** [reply]
I can't replicate the error. Can you try reinstalling the package? For example, use `pip install --upgrade --no-cache-dir --force-reinstall intensity-normalization==2.2.4`. Or create a new virtual environment/use conda. If that doesn't work, let me know.

**Reply by jcreinhold** [reply]
You probably need to add the path where the CLIs are installed to your `PATH` environment variable. When you pip install the package, I believe it'll list where the entrypoints are installed. Just prepend that directory to `PATH`; perhaps set it in .zshrc or whatever, so you don't have to set it each time you open a new terminal.

**Reply by jcreinhold** [reply]
FYI you should install brew, conda, do user installs of packages, or use a virtual environment. You don't want to be using or modifying the system Python.

**Comment by Jkgarciam** [comment]
Thank you I fixed the problem creating a  virtual environment.
Best, 

Josefa
